riptide-spring-boot-starter/src/main/java/org/zalando/riptide/spring/HttpClientFactoryBean.java

```diff
@@ -22,7 +23,7 @@
 import static java.lang.String.format;
 import static org.apache.http.conn.ssl.SSLConnectionSocketFactory.getDefaultHostnameVerifier;
 
-class HttpClientFactoryBean implements FactoryBean<HttpClient> {
+class HttpClientFactoryBean extends AbstractFactoryBean<CloseableHttpClient> {
```

Comment:
needs a extended unit test now (coverage) :wink:

Reply:
Locally it's covered...

Reply:
The merge is also fine...